reasoner_pydantic/utils.py

```diff
@@ -51,6 +51,9 @@ class HashableSequence(
 
     __root__: List[ValueType] = list()
 
+    def __contains__(self, v):
+        return v in self.__root__
```

Comment:
Should this call __contains__ ? I'm not sure it matters since __root__ is a List but maybe?

Reply:
I don't think it matters. From my minimal research, it seems like you shouldn't ever really call `__contains__` directly, it just enables you to call `in`.